rpc/src/server.rs

```diff
@@ -88,8 +88,10 @@ impl RpcServer {
     ) -> Result<SocketAddr, AnyError> {
         let stream_config = StreamServerConfig::default()
             .with_keep_alive(true)
-            .with_channel_size(4)
```

Comment:
Why should with_channel_size be changed to 8, and what impact will it have?
```
impl Default for StreamServerConfig {
    fn default() -> Self {
        Self {
            channel_size: 8,
            pipeline_size: 1,
            keep_alive: false,
            keep_alive_duration: Duration::from_secs(60),
            ping_interval: Duration::from_secs(19),
            shutdown_signal: future::pending().boxed().shared(),
        }
    }
}
```

Reply:
it's a channel size used in session:
```rust
   let (tx, mut rx) = channel(config.channel_size);
    let session = Session {
        id: SESSION_ID.fetch_add(1, std::sync::atomic::Ordering::Relaxed),
        raw_tx: tx,
    };
```

if the size is larger, the channel may hold more temporary messages from requests before the backend functional code handles it, but of course need more memory, it will only make a difference in a very high load scenario.

We don't have a specific reason to make it `4`, so let's use the default?